js/models/addressBook_model.js

```diff
@@ -13,9 +13,56 @@ app.factory('AddressBook', function()
 					}
 				}
 				return undefined;
+			},
+
+			sharedWith: {
+				users: [],
+				groups: []
 			}
 
 		});
 		angular.extend(this, data);
+		angular.extend(this, {
+			owner: data.url.split('/').slice(-3, -2)[0]
```

Comment:
Yup this is how calednar does it: https://github.com/owncloud/calendar-rework/blob/master/js/app/models/calendarmodel.js#L107